kotlin-jvm/src/main/kotlin/playground/StateMachine.kt

```diff
@@ -0,0 +1,90 @@
+@file:Suppress("PackageDirectoryMismatch")
+
+package playground.statemachine
+
+import com.tinder.StateMachine
+import playground.shouldBe
+
+/**
+ * Tinder/StateMachine - A kotlin DSL for finite state machine
+ *
+ * - [Tinder/State machine: Github](https://github.com/Tinder/StateMachine)
```

Comment:
Your state diagram is very nice. 
Doesn't it work in IntelliJ to add markdown syntax for images in the kdoc? That would be great. 

`States diagram:
![image](https://user-images.githubusercontent.com/6810202/95701244-73211700-0c1f-11eb-95f1-ce507b25a3bb.png)`

Reply:
I am not sure if there is a markdown for images but I have added the image link. 😉